asserts/fetcher.go

```diff
@@ -60,6 +63,34 @@ func NewFetcher(trustedDB RODatabase, retrieve func(*Ref) (Assertion, error), sa
 	}
 }
 
+type SeqFetcher interface {
```

Comment:
this probably needs a long name like SequenceFormingFetcher, also shouldn't this interface include Fetcher too?

Reply:
Good points, fixed!